tasks/Functions-client/setAutoRequest.js

```diff
@@ -1,15 +1,21 @@
+const { buildRequestCBOR, SecretsManager } = require("@chainlink/functions-toolkit")
+
 const { types } = require("hardhat/config")
 const { networks } = require("../../networks")
 const { getRequestConfig } = require("../../FunctionsSandboxLibrary")
-const { generateRequest } = require("./buildRequestJSON")
-const { RequestStore } = require("../utils/artifact")
-const { deleteGist } = require("../utils/github")
 const path = require("path")
 const process = require("process")
 
-task("functions-set-auto-request", "Updates the Functions request in a deployed AutomatedFunctionsConsumer contract")
+task(
+  "functions-set-auto-request",
+  "sets the CBOR-encoded Functions request in a deployed AutomatedFunctionsConsumer contract"
+)
   .addParam("contract", "Address of the client contract")
   .addParam("subid", "Billing subscription ID used to pay for Functions requests", undefined, types.int)
+  .addParam(
```

Comment:
This should be an optional param in case the user has specified `Location.remote` in Functions-request-config.js

Reply:
I think its best to just illustrate DON Hosted as that is the more secure and preferred approach.